app/assets/stylesheets/arclight/modules/hierarchy.scss

```diff
@@ -36,6 +36,13 @@
     line-height: 1.25;
   }
 
+  .al-document-abstract-or-scope-heading {
+    color: #636c72;
```

Comment:
This is probably my doing, since I gave that example in the ticket. I think `$gray-light;` might be it.